Dockerfile

```diff
@@ -1,11 +1,31 @@
-FROM golang:1.18 as workspace
+# dummy value for linux builds
+ARG WINBASE=scratch
```

Comment:
I prefer combined Dockerfiles as long as they aren't too complex, because people forget to keep Dockerfiles in sync (only few people seem to care about Windows). Yet, I understand your point of view and it's ok to split them.
I think this is something which can be changed later, back and forth, because the "interface" towards the user is the published image in a registry. So, no hard feelings from my side.